dojo.py

```diff
@@ -313,257 +251,51 @@ def load_state(self, db_sql):
-        room_name = Office(room_name)
-        return room_name
-    else:
-        room_name = LivingSpace(room_name)
-        return room_name
-
-def create_person(person_name, person_type, wants_accommodation):
-
-    if person_type == 'fellow' and wants_accommodation =='y':
-        person_name = Fellow(person_name, 'y')
-        print(" wants accommodation")
-        return person_name
-
-    elif person_type == 'fellow' and wants_accommodation=='n':
-        person_name = Fellow(person_name)
-        print(' doesn\'t want an accommodation and specified properly ')
-        return person_name
-    elif person_type == 'fellow' and wants_accommodation!='n':
-        person_name = Fellow(person_name)
-        print(' not added to living Space valid argument is "y" or "yes"')
-        return person_name
-    elif person_type == 'staff' and wants_accommodation=='n':
-        # get instance of the staff class and pass the appropriate arguments to db/data_structure
-        person_name = Staff(person_name)
-        print("staff added with the appropriate argument for wants_accommodation")
-        return person_name
-
+def check_room_size(room):
+    if len(room.occupants) < room.max_size:
+        return True
     else:
-        person_name = Staff(person_name)
-        print("Living rooms are only available for fellows ")
-
-#function to check whether a room is full
-
-#add a person to a room_type (office or living)and increase
-#the number of people there and check if it's
-#less than (4 for living or 6 for offices)
-#function to add a person to a room that is not full
-def add_person_to_room(person_name, wants_accommodation):
-    if wants_accommodation == 'n':
-        print(Dojo.app_session['room'])
-        #get an office that is not full and then append the person to the occupants
-        #add to office to instance of office if not full
-        #loop through the offices and add to one that is not full
-<<<<<<< HEAD
-        for index, value in enumerate(app_session['room']):
-            if(value['room_type'] =='office' and (value['room_size'] < value['max_occupants'])):
-=======
-        for index, value in enumerate(Dojo.app_session['room']):
-            print(value)
-            if(value['room_size'] < value['max_occupants']):
->>>>>>> ladi
-                value['room_occupants'].append(person_name)
-                value['room_size'] += 1
-                print(value['room_occupants'])
-                print(value)
-                return value['room_name']
-                break
-        else:
-            print("well I think we need more offices")
-            return ""
+        return False
+
+def get_available_room(purpose):
+    rooms = Dojo.all_rooms
+    available_rooms = []
+    for room in rooms:
+        if (room.purpose == purpose) and (len(room.occupants) < room.max_size):
+            available_rooms.append(room)
+    if available_rooms:
+        room = random.choice(available_rooms)
+        return room
     else:
```

Comment:
@andela-hoyeboade I have noted the points you raised. I will work on them and would require feedback afterwards. thank you.